### ISSUE TITLE
sources.bib not found

### ISSUE THREAD

**Issue by MiltosD** [glottolog/glottolog-cldf#18]
It seems that this link is broken..

[https://github.com/glottolog/glottolog-cldf/blob/master/cldf/sources.bib](https://github.com/glottolog/glottolog-cldf/blob/master/cldf/sources.bib)

**Comment by xrotwang** [comment]
Yes, sorry. Where did you find this link? At https://github.com/glottolog/glottolog-cldf/blob/master/cldf/README.md ?

Since the dataset contains now **all** Glottolog references linked to at least one languoid, the BibTeX file got quite big (~200MB), thus needed to be zipped, It's at https://github.com/glottolog/glottolog-cldf/blob/master/cldf/sources.bib.zip

**Comment by MiltosD** [comment]
So, it is intended that the sources.bib is now a zipped archive. I found the link there yes. I am using pycldf package which searches for https://github.com/glottolog/glottolog-cldf/blob/master/cldf/sources.bib, but I guess this functionality is now broken. Do you plan to release sources.bib as a zipped archive from now on?

**Comment by xrotwang** [comment]
If you upgrade `pycldf` it should work as is, see https://github.com/cldf/pycldf/blob/master/CHANGELOG.md#1340---2022-12-05

**Comment by xrotwang** [comment]
> Do you plan to release sources.bib as a zipped archive from now on?

Yes. `sources.bib.zip` is nicely below GitHub's "large file warning" limit of 50MB, while `sources.bib` would be above the hard size limit of 100MB. And I consider this solution still more comfortable than distributing the bib file as separate Zenodo upload, or not using the GitHub-Zenodo bridge anymore.

**Comment by MiltosD** [comment]
Thank you very much!

**Comment by xrotwang** [comment]
fixed now
